.github/workflows/ci.yml

```diff
@@ -39,15 +39,7 @@ jobs:
     steps:
       - uses: dtolnay/rust-toolchain@stable
 
-      - uses: actions/cache@v3
-        with:
-          path: |
-            ~/.cargo/bin/cargo-deny
-            ~/.cargo/.crates.toml
-            ~/.cargo/.crates2.json
-          key: ${{ runner.os }}-v2-cargo-deny-locked-0.8.5
-
-      - run: cargo install cargo-deny --locked --vers "0.8.5"
+      - uses: taiki-e/install-action@cargo-deny
```

Comment:
For deny, I think it would make more sense to use Embark's cargo-deny action directly?

Reply:
I don't know. It does the same but with the docker overhead.

Reply:
> The maintainers of install-action also appear a little bit more responsive than the cargo-deny folks.

In my experience the cargo-deny folks have usually been pretty responsive? (I'm okay with using this instead, though.)

Reply:
I have no actual experience with either upstream so defer to you :-) My comment is based on a very unscientific look at the open issues/PRs across the two repos and shouldn't be weighed very heavily.